mono/mini/aot-compiler.c

```diff
@@ -4427,7 +4427,13 @@ method_has_type_vars (MonoMethod *method)
 static
 gboolean mono_aot_mode_is_full (MonoAotOptions *opts)
 {
-	return opts->mode == MONO_AOT_MODE_FULL;
+	return opts->mode == MONO_AOT_MODE_FULL || opts->mode == MONO_AOT_MODE_INTERP;
```

Comment:
no, that stuff is guarded by `mono_aot_mode_is_interp ()`.  Other then that, it should behave like `mono_aot_mode_is_full`.